scipy/stats/_stats_py.py

```diff
@@ -887,9 +887,9 @@ def tsem(a, limits=None, inclusive=(True, True), axis=0, ddof=1):
 
 
 def _moment_outputs(kwds):
-    moment = np.atleast_1d(kwds.get('moment', 1))
+    moment = np.atleast_1d(kwds.get('order', 1))
     if moment.size == 0:
-        raise ValueError("'moment' must be a scalar or a non-empty 1D "
+        raise ValueError("'order' must be a scalar or a non-empty 1D "
```

Comment:
I guess this could be misleading if you were already using `moment` and then switched to a bad input but it is probably unlikely enough not to worry about